packages/grid/x-data-grid/src/locales/ruRU.ts

```diff
@@ -79,33 +79,33 @@ const ruRUGrid: Partial<GridLocaleText> = {
   filterOperatorIsEmpty: 'пустой',
   filterOperatorIsNotEmpty: 'не пустой',
   filterOperatorIsAnyOf: 'любой из',
-  // 'filterOperator=': '=',
-  // 'filterOperator!=': '!=',
-  // 'filterOperator>': '>',
-  // 'filterOperator>=': '>=',
-  // 'filterOperator<': '<',
-  // 'filterOperator<=': '<=',
+  'filterOperator=': '=',
+  'filterOperator!=': '!=',
+  'filterOperator>': '>',
+  'filterOperator>=': '>=',
+  'filterOperator<': '<',
+  'filterOperator<=': '<=',
 
   // Header filter operators text
-  // headerFilterOperatorContains: 'Contains',
-  // headerFilterOperatorEquals: 'Equals',
-  // headerFilterOperatorStartsWith: 'Starts with',
-  // headerFilterOperatorEndsWith: 'Ends with',
-  // headerFilterOperatorIs: 'Is',
-  // headerFilterOperatorNot: 'Is not',
-  // headerFilterOperatorAfter: 'Is after',
-  // headerFilterOperatorOnOrAfter: 'Is on or after',
-  // headerFilterOperatorBefore: 'Is before',
-  // headerFilterOperatorOnOrBefore: 'Is on or before',
-  // headerFilterOperatorIsEmpty: 'Is empty',
-  // headerFilterOperatorIsNotEmpty: 'Is not empty',
-  // headerFilterOperatorIsAnyOf: 'Is any of',
-  // 'headerFilterOperator=': 'Equals',
-  // 'headerFilterOperator!=': 'Not equals',
-  // 'headerFilterOperator>': 'Greater than',
-  // 'headerFilterOperator>=': 'Greater than or equal to',
-  // 'headerFilterOperator<': 'Less than',
-  // 'headerFilterOperator<=': 'Less than or equal to',
+  headerFilterOperatorContains: 'содержит',
+  headerFilterOperatorEquals: 'равен',
+  headerFilterOperatorStartsWith: 'начинается с',
+  headerFilterOperatorEndsWith: 'заканчивается на',
+  headerFilterOperatorIs: 'равен',
+  headerFilterOperatorNot: 'не равен',
+  headerFilterOperatorAfter: 'больше чем',
+  headerFilterOperatorOnOrAfter: 'больше или равно',
+  headerFilterOperatorBefore: 'меньше чем',
+  headerFilterOperatorOnOrBefore: 'меньше или равно',
+  headerFilterOperatorIsEmpty: 'пустой',
+  headerFilterOperatorIsNotEmpty: 'не пустой',
+  headerFilterOperatorIsAnyOf: 'Is any of',
+  'headerFilterOperator=': '=',
+  'headerFilterOperator!=': '!=',
+  'headerFilterOperator>': '>',
+  'headerFilterOperator>=': '>=',
+  'headerFilterOperator<': '<',
+  'headerFilterOperator<=': '<=',
```

Comment:
Can we either translate them or leave them as-is for someone else to translate?
P.S these values need to be textual as they'll be shown as helper text in [header filters](https://mui.com/x/react-data-grid/filtering/header-filters/).
```suggestion
  // headerFilterOperatorIsAnyOf: 'Is any of',
  // 'headerFilterOperator=': 'Equals',
  // 'headerFilterOperator!=': 'Not equals',
  // 'headerFilterOperator>': 'Greater than',
  // 'headerFilterOperator>=': 'Greater than or equal to',
  // 'headerFilterOperator<': 'Less than',
  // 'headerFilterOperator<=': 'Less than or equal to',
```